src/scipp/experimental/plotting/wrappers.py

```diff
@@ -16,13 +16,26 @@ def _to_data_array(obj):
         out = Variable(dims=dims, values=out)
     if isinstance(out, Variable):
         out = DataArray(data=out)
+    out = out.copy(deep=False)
     for dim, size in out.sizes.items():
         if dim not in out.meta:
             out.coords[dim] = arange(dim, size)
     return out
 
 
-def plot(obj: Union[VariableLike, Dict[str, VariableLike]]) -> Figure:
+def plot(obj: Union[VariableLike, Dict[str, VariableLike]],
+         aspect: str = 'auto',
+         cbar: bool = True,
+         crop: Dict[str, Dict[str, Variable]] = None,
+         errorbars: bool = True,
+         grid: bool = False,
+         mask_color: str = 'black',
+         norm: str = 'linear',
```

Comment:
```suggestion
         norm: Literal['linear', 'log'] = 'linear',
```
and `from typing import Literal`.

Same comment for a couple of other arguments.

Reply:
done